cmd/root.go

```diff
@@ -105,7 +108,20 @@ func start() {
 		Repository: repo,
 	})
 
-	server := server.NewServer(opt, service)
+	// Init graphql: load all schema and connect to services
+	resolver.Init(
+		resolver.WithServices(service),
+	)
+	schema, err := graphql.NewSchema(graphql.SchemaConfig{
+		Query: graphql.NewObject(query.GetQuerySchema()),
+		// Mutation: graphql.NewObject(mutation.GetMutationSchema()),
+	})
+	if err != nil {
+		logrus.Fatalf("Failed to create schema for the graphql: %s", err)
+		return
+	}
+
+	server := server.NewServer(opt, service, schema)
```

Comment:
would be better if graph-ql can be configurable as not mandatory

Reply:
@triardn just reminder for this :)